cmd/cortex/main.go

```diff
@@ -139,6 +143,12 @@ func main() {
 		prometheus.MustRegister(resourceWatcher)
 	}
 
+	tracer, err := loki.NewTracer(loki.DefaultConfig)
```

Comment:
I've literrally copied the exact way prometheus does this: https://github.com/tomwilkie/loki/blob/master/pkg/config/config.go

Are we talking cross purposes?